src/cdef_tmpl_16.rs

```diff
@@ -7,7 +7,7 @@ extern "C" {
     static dav1d_cdef_directions: [[int8_t; 2]; 12];
 }
 
-#[cfg(feature = "asm")]
+// #[cfg(feature = "asm")]
```

Comment:
I should have been clearer. I made this commit quickly as a workaround; don't have a strong opinon on what the best resolution is. That said, it might be worthwhile to stay close to the C code for the moment. AFAICT, the C code unconditionally [defines](https://github.com/memorysafety/rav1d/blob/d35c4ed2edd48026508cd77cf35f674ae75c44ac/src/cpu.h) and [uses](https://github.com/memorysafety/rav1d/blob/d35c4ed2edd48026508cd77cf35f674ae75c44ac/src/cpu.h#L54) these variables. .

Reply:
I think I was able to do what you suggest with some more compile-time guards.